### ISSUE TITLE
network todos

### ISSUE THREAD

**Issue by kwagstyl** [imaginelab/text2brain#29]
To improve the network:
- nonlinear transformation layer from embedding to internal latent space  to reorganise language embedding
- loss BCE
- output logits vs sigmoid
- check that the deconvolutions work - ?checkerboard artefact
- deep supervision

- other ways to improve existing architecture
- ?FILM layers
- augmentations - language (introduce those Alice) and mask
- intermediate layer to learn things
- skip connections
- symmetrical atlases need handling
- giving the MNI as an input (essential unet with additional info) (important step for native space)
- improved validation metrics (DICE), images?
-
